lib/messages.js

```diff
@@ -27,7 +27,7 @@ export const NO_STAGED_FILES = `${info} No staged files found.`
 export const NO_TASKS = `${info} No staged files match any configured task.`
 
 export const skippingBackup = (hasInitialCommit) => {
-  const reason = hasInitialCommit ? '`--no-stash` was used' : 'there’s no initial commit yet'
+  const reason = hasInitialCommit ? '`--no-stash` was used' : "there's no initial commit yet"
```

Comment:
I wonder why you changing the apostrophe to a seconds sign. Typographically speaking the apostrophe is a correct punctuation here. Let's leave as is please everywhere.

Reply:
My editor was complaining about them. Let me remove the commit, no problem.

Reply:
Updated.